src/models/player.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Player extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      this.hasMany(models.Play, {
+        foreignKey: 'player',
+        onDelete: 'CASCADE'
+      });
+      this.hasMany(models.Match, {
```

Comment:
esta debería ser `hasAndBelongsToMany` para cerrar la relación desde este lado https://sequelize.org/docs/v6/core-concepts/assocs/ (Many to Many associations)

Reply:
tu deci acá crear un nuevo atributo en match que sea winner?